### ISSUE TITLE
Starting the drag on a link misbehaves on IE

### ISSUE THREAD

**Issue by Ceremony64** [metafizzy/flickity#581]
Shitty IE (and probably Edge - can't test, I'm on Win7) has an issue, if you drag a carousel when you start on a link.

If you start dragging with the cell/link, IE tries to move the content of the link, while the carousel freezes right after the drag started. Once you let go of the mousebutton, the carousel still moves according to your movements, until you click again.

This is especially annoying if you use cells that are basically "all link".

**Test case:** https://jsfiddle.net/jt5vswsx/

**Comment by desandro** [comment]
Thanks for reporting this issue. I'll have to investigate.

**Comment by yeahdino** [comment]
+1 

This is also the case for <img> elements in IE11.

**Same test case but with <img>:** https://jsfiddle.net/jt5vswsx/4/

Thank you for all your effort in developing Flickity. Excellent plugin.

**Comment by Ceremony64** [comment]
For those running into this issue and do require that ancient IE support (like me), here is a quick hack:
```javascript
$('#carousel').flickity({
  // options and stuff
}).find('a').on('mousedown', function (e) {
  e.preventDefault();
  return false;
});
```

**This cancels the mousedown event** which causes the drag&drop ~~feature~~ bug of IE to kick in. Simply clicking on links should still work just fine!

P.S. For those running vanilla flickity, here's the code:
```javascript
var carousel = document.querySelector('#carousel');
new Flickity(carousel, {
  // options
});
Array.prototype.forEach.call(carousel.getElementsByTagName("a"), function(el){
  el.addEventListener('mousedown', function(e) {
    e.preventDefault();
    return false;
  });
})
```

**Comment by Ceremony64** [comment]
Nevermind my previous comment! This issue has been fixed in 2.0.10 probably by fixing #567 

Do **not** use the code from my previous post on 2.0.10 and beyond as this will break slideshows on linked content.

**Comment by desandro** [comment]
Thanks for following up!
